pkg/mime/mime.go

```diff
@@ -47,8 +47,7 @@ func Detect(isDir bool, fn string) string {
 		return defaultMimeDir
 	}
 
-	ext := path.Ext(fn)
-	ext = strings.TrimPrefix(ext, ".")
+	ext := strings.ToLower(strings.TrimPrefix(path.Ext(fn), "."))
```

Comment:
This is not needed because line 55 is doing a comparison with lowercase

Reply:
Ah! I "jumped" on it to try and keep those parts in sync, thanks for noting. I'll close the PR.